projects/canopy/src/lib/filter-buttons/filter-button/filter-button.component.scss

```
@@ -0,0 +1,99 @@
+@import '../../../styles/mixins';
+
+.lg-filter-button__input {
+  @include lg-visually-hidden;
+}
+
+.lg-filter-button__label {
+  display: inline-block;
+  max-height: var(--space-lg);
+  border: solid var(--border-width) var(--filter-button-bg-color-active);
+  border-radius: var(--space-md);
+  padding: var(--space-xxxs) var(--space-sm) var(--space-xxxs) var(--space-sm);
+  background: transparent;
+  font-size: var(--text-fs--8-size);
+  margin-bottom: var(--space-xs);
+  cursor: pointer;
+
+  .lg-icon {
+    vertical-align: sub;
+    margin-right: var(--space-xxxs);
+  }
+
+  .filter-button__icon-selected {
```

```suggestion
  .lg-filter-button__icon-selected {
```
It's probably good practice to add the `lg` prefix to these. We do this mostly as `ViewExncapsulation` is currently set to `None` which means there is a risk of class clashing where people use a different library alongside Canopy.  Unlikely with such specific names, but it does keep it consistent.

---

Will keep in mind in the future.

---

Yes, I think it's better to add the prefix so that it's consistent with what we've done until now 👍

---

Done.